Changed the default configuration for `pytest` so that if a test is marked
as expected to fail actually passes, then that test will issue an error.
